src/pkg/runner/actions.go

```diff
@@ -163,7 +181,9 @@ func (r *Runner) performZarfAction(action *zarfTypes.ZarfComponentAction) error
 
 	cfg := actionGetCfg(zarfTypes.ZarfComponentActionDefaults{}, *action, r.TemplateMap)
 
-	if cmd, err = actionCmdMutation(cmd); err != nil {
+	configWrapper := Config{cmdPrefix: config.CmdPrefix}
+
+	if cmd, err = actionCmdMutation(cmd, ZarfUtils{}, configWrapper); err != nil {
```

Comment:
Another suggestion. Instead of directly creating the `ZarfUtils{}` use a `New` to return the interface

```go
func NewZarfUtils() Utils {
	return &ZarfUtils{}
}
```

Reply:
addressed in latest commit